tests/framework/cli/test_starters.py

```diff
@@ -68,25 +69,6 @@ def _make_cli_prompt_input_without_name(
     return "\n".join([add_ons, repo_name, python_package])
 
 
-def _convert_addon_names_to_numbers(selected_addons: str):
```

Comment:
this function is actually more explicit than the `_get_addons_from_cli_input`. It is only used in test originally, I've rename the original function and remove this from test.